ci-operator/step-registry/mirror-images/by-oc-adm-in-bastion/mirror-images-by-oc-adm-in-bastion-commands.sh

```diff
@@ -25,7 +25,7 @@ function run_command() {
 }
 
 mirror_output="${SHARED_DIR}/mirror_output"
-pull_secret_filename="new_pull_secret"
+pull_secret_filename="$(<"${SHARED_DIR}/cluster_name")_pull_secret"
```

Comment:
Do we really need to change the filename? The file name is for naming the remote pull secret, the remote pull secret would be copied from local, after that, the file is used to run `oc adm release mirror` in the remote bastion host.

Reply:
Yes, not needed. I was thinking in terms of 2 disconnected jobs running in parallel and leading to overwriting the pull secret on the aux host. But the pull secret is the same for every cluster.